js/binance.js

```diff
@@ -1913,7 +1919,13 @@ module.exports = class binance extends Exchange {
         } else if ((type === 'margin') || (marginMode === 'cross')) {
             method = 'sapiGetMarginAccount';
         } else if (type === 'savings') {
-            method = 'sapiGetLendingUnionAccount';
+            const lockedProduct = this.safeStringUpper (params, 'lockedProduct');
+            if (lockedProduct !== undefined) {
+                request['product'] = lockedProduct;
+                method = 'sapiGetStakingPosition';
```

Comment:
@sc0Vu noticed this endpoint is under the `staking` category, should we create a new `staking` type? https://binance-docs.github.io/apidocs/spot/en/#get-staking-product-list-user_data